deploy/f2f-spec.yaml

```diff
@@ -0,0 +1,864 @@
+      required:
+        - client_id
+        - request
+      properties:
+        client_id:
+          type: string
+          example: abcdefgh
+          description: >-
+            the OAuth client identifier which has been registered with F2F
+        request:
+          type: string
+          example: eyJhbGciOiJSU0EtT0FFUC0yNTYiLCJlbmMiOiJBMjU2R0NNIn0.AISIEvimTkm_1gfG9gQA5btB7lHu2riU90KI-UyuH35CABCJk_KKUTsptasmNG5n6Zo6jL8vlMrj3y4A44nDrnTeH_OGLByerCd-EKNx9EZrplcsQ8VM8dip7LCUQy10OGH6ggoMGq9TbiC7lP_8md8UgvcNzd_zm6q6Ade5zd_sUzTWNSX3q6lVfpkpuMdj-2l18zk-yJEvaDArP7U1VBT8531ixxW1lUKflTHlyC-t6B-45R04o3KubhVb-_gVt7KKXo7UQUY-wmHiTUGsxWb3xrROKs6tv4Ug7Oi1MVngesL8CahjyLamvk-tdS1ji2rxn47Xs2Q21FQf3cHPtQ.9bNdWPzw2Cf6uyR_.VVj96Ejcc84yKv3gfHY6Nond9FJrQ-AoQ7pQSlmnzicoJ7hpHwyHUuNiSJNO4q631VKAblNbdprAA11PcYL3f_hcocxwORR8e3tQlDG9TFn6lxg3l9M34ccJrkD51ippIpsChQhIFqS2hU5W9BFyOPdpQhlhtc7R-d0zqbylh-z-snzxRG7Q-hVsBQpOzOLMDWcJNOCHlF8kZjp4dlWYFj3Yl_9HAa-hrY-DpoxdkvXVIrBdJarD77Bwwxo8y1IWEgCuDA8t5ZmrZMY9fxoufQg9jW5D7hB3qysgSo3S-6SFyt3rP_WD5ecezRrTCY-voDeWBtfdwuKo5UUaPidN4ut1n8SXAdoANxgbuEhyABjvMESi3AnWwZRx8yh3xEMVSwZ8kFQ4cVeBW2hrkiSXMjL1Wtz6nhhY9itFcmzBU4JMvoYfdf-ZZP-F4ph_Xh9CRS7rBzmrgUt81LahzYfCY91lS93qT7r3Wrh0OfIb0vvIkVUebDdCZw25PoG2oREty4k59z7j9ho8VqbtSS6_kpJGWx8Sikut1I-bVduk79Vf1MwEGxSRvZ_pyZHxxZDe19GPfamhctFXMcebyJoT2zwWhcW4AWFSUtm2JI36sZhFcsdRIRBfSjJ4MNXdlrHdOyhvo8SwTWpfO5lU9yyF8lANbXxjdnRSV6sR1iaBs_16slbWJX3WC6RmC76x8MlHn1fLBK_lU_IeZChZqLCwqUC3CXbk17OkTkdeCqFyZSS5rlfm-EiPvam6iW51tA-ywxrmUjkCDE_k-zZIZTVVSg.3YAfgcQ4ZkPtYPNvOzWCLQ
+          description: >-
+            an JWT which has been signed with the clients signing private key and encrypted with the F2F public
+            encryption key.
+    IssueTokenRequest:
+      type: object
+      additionalProperties: false
+      required:
+        - grant_type
+        - code
+        - redirect_uri
+      properties:
+        grant_type:
+          type: string
+          example: authorization_code
+          description: >-
+            OAuth grant type. At present the only supported grant type is `authorization_code`.
+          enum:
+            - authorization_code
+        code:
+          type: string
+          format: uuid
+          example: 33333333-3333-3333-3333-333333333333
+          description: >-
+            `authorization_code` as issued by POST /authorizationCode.
+        redirect_uri:
+          type: string
+          example: https://www.example.com/receiveToken
+          description: >-
+            the OAuth client identifier which has been registered with CI CRI
+        client_assertion:
+          type: string
+          example: abcdefgh
+          description: >-
+            Client assertion (use for client credentials flow, not supported here)
+        client_assertion_type:
+          type: string
+          example: jwt_bearer
+          description: >-
+            Client assertion type (use for client credentials flow, not supported here)
+    ClaimedIdentityRequest:
```

Comment:
we dont have ClaimedIdentityRequest in F2F

Reply:
[Resolved](https://github.com/alphagov/di-ipv-cri-f2f-api/pull/9/commits/071a44f2076a7ca46d284a45dd1192a3422e1143)